src/functionalTest/groovy/com/autonomousapps/android/projects/AbstractVariantProject.groovy

```diff
@@ -46,14 +45,14 @@ final class VariantProject extends AbstractProject {
     return project
   }
 
-  private List<Plugin> plugins = [
+  protected final List<Plugin> plugins = [
```

Comment:
Those private property weren't seen in the closure
```
builder.withRootProject {}
```
As I understand Closure works like that I see actual class not an abstract one because of that I marked those props protected